source/Playnite.FullscreenApp/Controls/ListBoxEx.cs

```diff
@@ -21,6 +21,9 @@ public ListBoxEx() : base()
         {
             SelectionChanged += ListBoxEx_SelectionChanged;
             this.GotFocus += ListBoxEx_GotFocus;
+
+            // Set to always left-to-right

```

Comment:
Why is this being set always left-to-right?

Reply:
Actually I thought it made a difference in the up-side-down symptom, but I removed it and it didn't. So reverted.